### DISCUSSION TITLE
Matching Rate nextclade

### DISCUSSION THREAD

**Discussion by mcnaber** [nextstrain/nextclade#995] [Help]
Hello there, 

I am working to edit the required matching rate for nextclade. My goal is to try and lower it to see a larger portion of my data sets and/or include alignments in my final result even if they don't meet the 0.300 minimum matching rate. Is there any way to edit this feature? I can't seem to find a command that controls this. Thank you.

**Comment by ivan-aksamentov** [comment]
Hi @mcnaber,

If you are using CLI, you can use `--min-match-rate` argument to set minimum seed matching rate. Read more in the built-in help:

```bash
nextclade run --help
```
